Content/ua_20200713_Feats-2020.js

```diff
@@ -8,4 +8,306 @@ SourceList["UA:F20"] = {
 	group : "Unearthed Arcana",
 	url : "https://media.wizards.com/2020/dnd/downloads/UA2020_Feats.pdf",
 	date : "2020/07/13"
+};
+
+FeatsList["magic initiate [artificer]"] = {
```

Comment:
This feat is named Artificer Initiate in the source.

Reply:
In the main document, the Magic Initiate feats are named in this manner to keep them grouped together. I figured I would stick with overall convention as to avoid confusion when someone is looking for the feat.

Reply:
Those are variants of the 'Magic Initiate' feat and the class name only shows when you choose a variant.

I think it's best to stick with the feat as named in the source document.

Reply:
Change made